autoemulate/core/tuner.py

```diff
@@ -101,7 +101,7 @@ def run(  # noqa: PLR0913
         # Initialize retries and maximum number of retries for consecutive failed tuning
         # iterations that would indicate that no model params is working for the
         # given dataset/model and that the tuning process should be stopped
-        retries, max_retries = 0, 100
+        retries, max_retries = 0, 5
```

Comment:
Perhaps it's worth adding an extra comment explaining the implications of the value?
```suggestion
        # max_retries is set to 5 so that if the model or available tune_params do not
        # enable model fitting, the tuner can exit. If this is an unlikely false
        # positive, the whole tuning process can be retried
        retries, max_retries = 0, 5
```